aviary/docs/user_guide/propulsion.ipynb

```diff
@@ -145,7 +145,7 @@
     "ae = Aircraft.Engine\n",
     "\n",
     "required = (ae.DATA_FILE, ae.SCALE_PERFORMANCE, ae.IGNORE_NEGATIVE_THRUST, ae.GEOPOTENTIAL_ALT, ae.GENERATE_FLIGHT_IDLE)\n",
-    "# ********NOTE******** #\n",
+    "# ****NOTE**** #\n",
     "# these do not match, should the docs be updated?\n",
     "# check_value(required,required_options)"
```

Comment:
@jkirk5

Reply:
so should I update the docs to clarify that this is not a list of the minimum set:
![image](https://github.com/user-attachments/assets/a949d005-7654-4c4a-87db-7dd3d9e71720)

Reply:
On the actual Aviary side: you need to define something related to every bullet point on the list, but not necessarily every single variable called out. In that way, it is truly is the minimum variable list.

On the glue side: we should check all of the variables called out here exist within the code

Reply:
Should `Aircraft.Engine.DATA_FILE` be added to `required_options` in `engine_deck.py`?

Reply:
@jkirk5

Reply:
Yes, sorry for late reply